src/test/java/io/supertokens/test/passwordless/PasswordlessGetUserTest.java

```diff
@@ -61,14 +63,21 @@ public void beforeEach() {
      */
     @Test
     public void getUserByIdWithEmail() throws Exception {
-        TestingProcessManager.TestingProcess process = startApplicationWithDefaultArgs();
+        String[] args = { "../" };
+
+        TestingProcessManager.TestingProcess process = TestingProcessManager.start(args);
+        assertNotNull(process.checkOrWaitForEvent(ProcessState.PROCESS_STATE.STARTED));
+
+        if (StorageLayer.getStorage(process.getProcess()).getType() != STORAGE_TYPE.SQL) {
+            return;
+        }
 
         PasswordlessStorage storage = StorageLayer.getPasswordlessStorage(process.getProcess());
         UserInfo user = null;
 
         Passwordless.ConsumeCodeResponse consumeCodeResponse = createUserWith(process, EMAIL, null);
 
-        user = storage.getUserById(consumeCodeResponse.user.id);
+        user = Passwordless.getUserById(process.getProcess(), consumeCodeResponse.user.id);
         assertNotNull(user);
         assertEquals(user.email, EMAIL);
```

Comment:
The test processes are not killed in these cases.

Reply:
roger.